index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+        <!-- navigation bar begins here -->
+            <a href="#"><h2 class="logo">Diana</h2></a>
+            <div class="maimenu">
+                <a href=""><img src="image/Union.svg" alt="union-icon"></a>
+            </div>
+    </nav>
+</header>
+<main>
+    <section id="flex">
+        <!--navigation bar ends  -->
+      <div class="main-section">
+            <div class="first-heading">
+              <h1>Hey There.<br>
+                  I'm Diana</h1>
+        </div>
+        <div class="secon-heading">
+            <h2>I’am a Software Developer</h2>
+        </div>
+        <div class="details">
+            <p>I can help you build a product, feature
+               website Look through some of my work
+               and experience! if you like what you see
+               and have a project you need coded
+               don't hesitate to contact me.
+            </p>
+        </div>
```

Comment:
Your Design is not showing well at the mobile size width (375px) 
For example: the paragraph is disappearing and there is a big white space at the bottom and your font size looks too large
![image](https://user-images.githubusercontent.com/87448628/185609091-f971662d-5ac5-40c0-8f27-bdd103b41b2b.png)
![image](https://user-images.githubusercontent.com/87448628/185609197-a901259c-9f0c-4acf-9dc0-73e6ecdc30d8.png)
although you have all the elements you need them to look good on a screen size of 375px

Kindly fix these issue 👍 😄